File: IdealAutomateScriptRecorder/ImageEntity.vb
Public Class ImageEntity
    Public Property ImageFile() As String
    Public Property Attempts() As Integer
    Public Property Occurrence() As Integer
    Public Property Sleep() As Integer
    Public Property RelativeX() As Integer
    Public Property RelativeY() As Integer
    Public Property Tolerance() As Integer
    Public Property UseGrayScale() As Boolean
    Public Sub New()
        Attempts = 1
        Occurrence = 1
        Sleep = 100
        RelativeX = 10
        RelativeY = 10
        Tolerance = 90
        UseGrayScale = False
    End Sub
End Class

File: ScriptGenerators/Window/WindowBalloonMultipleControls/WindowBalloonMultipleControls/ImageEntity.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace IdealAutomate.Core
{
    public class ImageEntity
    {
        public string ImageFile { get; set; }
        public int Attempts { get; set; }
        public int Occurrence { get; set; }
        public int Sleep { get; set; }
        public int RelativeX { get; set; }
        public int RelativeY { get; set; }       
        public int Tolerance { get; set; }
        public bool UseGrayScale { get; set; }
        public ImageEntity()
        {
            Attempts = 1;
            Occurrence = 1;
            Sleep = 100;
            RelativeX = 10;
            RelativeY = 10;
            Tolerance = 90;
            UseGrayScale = false;
        }
    }
}
